### ISSUE TITLE
python version needed

### ISSUE THREAD

**Issue by vondele** [lucasart/c-chess-cli#45]
The build process seems to require python 3.7 or newer:
```
python3 make.py 
Traceback (most recent call last):
  File "make.py", line 14, in <module>
    version = subprocess.run(['git', 'show','-s','--format=%ci'], capture_output=True).stdout.split()[0].decode('utf-8')
  File "/usr/lib64/python3.6/subprocess.py", line 423, in run
    with Popen(*popenargs, **kwargs) as process:
TypeError: __init__() got an unexpected keyword argument 'capture_output'
```

**Comment by vondele** [comment]
using `stdout=subprocess.PIPE, stderr=subprocess.PIPE` instead of `capture_output=True` fixes it for me.

**Comment by lucasart** [comment]
fixed 2a2bdbd07ab3e85b65748853253b372864141f13

**Comment by ppigazzini** [comment]
I suggest to state the minimum python version supported.
Pyhton 3.5 is EOL since September 2020 and on Linux is easy to install any python distro/version with (pyenv)[https://github.com/pyenv/pyenv]
https://endoflife.date/python
